modules/opensearch/includes/opensearch.search.inc

```diff
@@ -99,16 +99,13 @@ function opensearch_search_search(TingSearchRequest $ting_query) {
     $query_parts[] = $ting_query->getRawQuery();
   }
 
-  if (!empty($simple_query_string = $ting_query->getSimpleQuery())) {
-    // Try to build the best search query based on the inputted string.
-    if (preg_match('/(^\d+$)|(^\d+:\d+$)|(=)/', $simple_query_string)) {
-      // The query is either ISBN, tid or CQL search expression.
-      $query_parts[] = $simple_query_string;
-    }
-    else {
-      // Add wildcard to the search string.
-      $query_parts[]  = _ting_search_quote($simple_query_string) . '*';
+  // Add a general quoted free text search.
+  if (!empty($free_text_query = $ting_query->getFullTextQuery())) {
+    $free_text_query = _ting_search_quote($ting_query->getFullTextQuery());
+    if ($ting_query->isFuzzy()) {
+      $free_text_query = '*' . $free_text_query . '*';
```

Comment:
This adds a wildcard before and after the query. The original implementation only had a wildcard after. Is this change intentional?

Reply:
Yes, intentional - but I should probably test how it actually affects the UX of the reference widget.

My thinking was that it makes more sense surround with the wildcard for a setting named "isFuzzy"